python/ray/train/_internal/backend_executor.py

```diff
@@ -111,6 +111,20 @@ def start(
             actor_cls_kwargs=train_cls_kwargs,
             placement_group=placement_group,
         )
+        # RLlib's LearnerGroup is also using BackendExecutor now and they don't
+        # pass in `TrialInfo`.
+        if self._trial_info:
+            worker_loc_str = [
+                f"{w.metadata.node_ip}:{w.metadata.pid}"
+                for w in self.worker_group.workers
+            ]
+            if self._trial_info.is_training:
```

Comment:
For the reasons mentioned in the PR description, I'd prefer we keep this simple for now and not include this training/tuning differentiation (i.e. remove all the `is_training` changes). 

For the Tune use case if the user wants to tie back the Trial name to this log I think they can just map the `pid` of the `BackendExecutor` themselves.

Reply:
ah we already have Trial name stuff currently :) 
piping `is_training` is to address your comment that for training case, trial name shouldn't be printed.
so what do we want the behavior to be if we literally don't know whether we are in training or tuning?

Reply:
I think it's fine to just print without the trial name!

Reply:
I actually prefer it to be option 1, it's more natural when everything is nicely located together.

Reply:
Let's start without it and add it as needed?

Reply:
done!